Missing dune-action-plugin dependency

Hi,

Your library looks extremely promising, but I struggled to install it.

It first complained about missing `/usr/local/opt/libpq/lib/postgresql/libpgport.a`:
```
(cd _build/default/src/typ && /usr/bin/cc -I /Users/paul/code/dialo/_opam/lib/ctypes -I /Users/paul/code/dialo/_opam/lib/ocaml -I/usr/local/opt/libpq/include/postgresql/server -o pgo_typ_gen2.exe pgo_typ.o /usr/local/opt/libpq/lib/postgresql/libpgport.a)
clang: error: no such file or directory: '/usr/local/opt/libpq/lib/postgresql/libpgport.a'
```
So I created a a symlink from `/usr/local/opt/libpq/lib/libpgport.a` (I don't know if something is wrong in my libpq install) and it then stopped at another issue:
```
File "src/fdw/dune", line 24, characters 23-41:
24 |  (libraries base stdio dune-action-plugin str))
                            ^^^^^^^^^^^^^^^^^^
Error: Library "dune-action-plugin" not found.
-> required by _build/default/src/fdw/pgo_fdw_build.exe
-> required by _build/install/default/bin/pgo-fdw-build
-> required by _build/default/pgo.install
-> required by alias install
(cd _build/default/src/api && /usr/bin/cc -O2 -fno-strict-aliasing -fwrapv -pthread -D_FILE_OFFSET_BITS=64 -O2 -fno-strict-aliasing -fwrapv -pthread -Werror=implicit-function-declaration -I/usr/local/opt/libpq/include/postgresql/server -fPIC -I. -g -I /Users/paul/code/dialo/_opam/lib/ocaml -I /Users/paul/code/dialo/_opam/lib/base -I /Users/paul/code/dialo/_opam/lib/base/base_internalhash_types -I /Users/paul/code/dialo/_opam/lib/base/caml -I /Users/paul/code/dialo/_opam/lib/base/shadow_stdlib -I /Users/paul/code/dialo/_opam/lib/bigarray-compat -I /Users/paul/code/dialo/_opam/lib/ctypes -I /Users/paul/code/dialo/_opam/lib/integers -I /Users/paul/code/dialo/_opam/lib/sexplib0 -I /Users/paul/code/dialo/_opam/lib/stdlib-shims -I . -I ../typ -o pgo_api.o -c pgo_api.c)
pgo_api.c:17:57: warning: format string is not a string literal (potentially insecure) [-Wformat-security]
  ereport(ERROR, (errcode(ERRCODE_SYNTAX_ERROR), errmsg(s)));
                                                        ^
/usr/local/opt/libpq/include/postgresql/server/utils/elog.h:158:37: note: expanded from macro 'ereport'
        ereport_domain(elevel, TEXTDOMAIN, __VA_ARGS__)
                                           ^~~~~~~~~~~
/usr/local/opt/libpq/include/postgresql/server/utils/elog.h:141:4: note: expanded from macro 'ereport_domain'
                        __VA_ARGS__, errfinish(__FILE__, __LINE__, PG_FUNCNAME_MACRO); \
                        ^~~~~~~~~~~
pgo_api.c:17:57: note: treat the string as an argument to avoid this
  ereport(ERROR, (errcode(ERRCODE_SYNTAX_ERROR), errmsg(s)));
                                                        ^
                                                        "%s",
/usr/local/opt/libpq/include/postgresql/server/utils/elog.h:158:37: note: expanded from macro 'ereport'
        ereport_domain(elevel, TEXTDOMAIN, __VA_ARGS__)
                                           ^
/usr/local/opt/libpq/include/postgresql/server/utils/elog.h:141:4: note: expanded from macro 'ereport_domain'
```
Adding `dune-action-plugin` to my dependencies fixed it but I guess it should be listed among pgo's dependencies given it's used in `/src/fdw`.

--- a/dune-project
+++ b/dune-project
@@ -7,7 +7,7 @@
 (package
  (name pgo)
  (synopsis "Build PostgreSQL extensions in OCaml")
- (depends dune dune-configurator ctypes)
+ (depends dune dune-action-plugin dune-configurator ctypes)
  (maintainers Andrey Popp <me@andreypopp.com>)
  (authors Andrey Popp <me@andreypopp.com>)
  (homepage https://github.com/andreypopp/pgo)

--- a/pgo.opam
+++ b/pgo.opam
@@ -8,6 +8,7 @@ homepage: "https://github.com/andreypopp/pgo"
 bug-reports: "https://github.com/andreypopp/pgo"
 depends: [
   "dune" {>= "2.9"}
+  "dune-action-plugin"
   "dune-configurator"
   "ctypes"
   "odoc" {with-doc}

